### PR TITLE
Adding Teams app Installation consent permission.

### PR DESCRIPTION
**Pull request by sweta-thapliyal**

### CODE REVIEW

**Review by AkJo — APPROVED** [review]

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -2307,6 +2307,10 @@ Taxonomy permissions are valid only on work or school accounts.
 | _TeamsAppInstallation.ReadForTeam_ | Read installed Teams apps in teams| Allows the app to read the Teams apps that are installed in teams the signed-in user can access. Does not give the ability to read application-specific settings.| Yes | No |
 | _TeamsAppInstallation.ReadWriteForTeam_ | Manage installed Teams apps in teams| Allows the app to read, install, upgrade, and uninstall Teams apps in teams the signed-in user can access. Does not give the ability to read application-specific settings.| Yes | No |
 | _TeamsAppInstallation.ReadWriteSelfForTeam_ | Allow the app to manage itself in teams| Allows a Teams app to read, install, upgrade, and uninstall itself to teams the signed-in user can access.| Yes | No |
+| _TeamsAppInstallation.ReadWriteAndConsentForChat_ | Manage installation and permission grants of Teams apps in chats| Allows the app to read, install, upgrade, and uninstall Teams apps in chats the signed-in user can access. Gives the ability to manage permission grants for accessing data for those specific.| Yes | No |
```
Comment (Lauragra) [inline concepts/permissions-reference.md:2310]: Please include a changelog entry to call out the addition of these new permissions. You can follow the guidance in the changelog topic: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-properties.

**Review by Lauragra — COMMENTED** [review]
Please see my review comment.

File: concepts/permissions-reference.md (hunk 1 above)
Comment (AkJo) [inline concepts/permissions-reference.md:2310]: for those specific chats

File: changelog/Microsoft.Teams.Core.json [hunk 2]
```diff
@@ -2262,6 +2262,24 @@
       "WorkloadArea": "Teamwork and communications",
       "SubArea": ""
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "46b0ea99-e62b-4cc6-85dd-9117a796c440",
+          "ApiChange": "Resource",
+          "ChangedApiName": "TeamsAppInstallation.ReadWriteAndConsentForChat.All,TeamsAppInstallation.ReadWriteAndConsentSelfForChat.All,TeamsAppInstallation.ReadWriteAndConsentForTeam.All,TeamsAppInstallation.ReadWriteAndConsentSelfForTeam.All,TeamsAppInstallation.ReadWriteAndConsentForChat,TeamsAppInstallation.ReadWriteAndConsentSelfForChat,TeamsAppInstallation.ReadWriteAndConsentForTeam,TeamsAppInstallation.ReadWriteAndConsentSelfForTeam",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:2270]: Please create a separate entry for each new permission. The ChangedApiName and Target fields are single valued. For details about Changelog entries, see this Wiki topic: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-properties

Also, are there API topics that use these new permissions that you need to update?

**Pull request closed without merging**